### PR TITLE
React bootcamp deliverable 1

### PR DESCRIPTION
**Pull request by juanhdezran88**
React bootcamp deliverable 1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,16 +10,28 @@
...
     "eject": "react-scripts eject",
     "lint": "eslint ./src --ext .js,.jsx",
     "lint:fix": "eslint ./src --ext .js,.jsx --fix"
   },
+  "jest": {
+    "coverageThreshold": {
+      "global": {
+      "branches": 60,
+      "functions": 60,
+      "lines": 60,
```
Comment (dianasaurio) [inline package.json:30]: awesome! 🌈

File: src/components/Card/index.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Image, CardWrapper } from './styled';
+
+const Card = ({title, imageURL, description, publishedDate}) => {
+  return (
+    <CardWrapper className="ui card" data-testid="card-wrapper">
+      <a className="image" href="/">
+        <Image style={{backgroundImage: `url(${imageURL})`}} />
```
Comment (dianasaurio) [inline src/components/Card/index.jsx]: Why are you using inline styles if Image is a styled component? You can just pass the background image as a prop.
https://styled-components.com/docs/basics#passed-props

File: src/components/Card/styled.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+const Image = styled.div`
+  display: block;
+  width: 100%;
+  height: 150px;
```
Comment (dianasaurio) [inline src/components/Card/styled.js]: I suggest you use relative units like rem. This helps you achieve a more responsive application. Ideally, you should only use pixels for font sizes, borders, and small stuff.

https://thecssworkshop.com/lessons/relative-units

File: src/components/Header/index.jsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+const Header = () => {
+  return (
+    <HeaderPanel data-testid="header">
+      <div className="ui icon button toggle-button">
+        <i className="bars icon"></i>
+      </div>
+      <SearchInput type="text" className="search-input" placeholder="Search..." />
+      <ControlPanel>
+        <Toggle className="ui toggle checkbox">
```
Comment (dianasaurio) [inline src/components/Header/index.jsx:12]: Not a big fan of mixing styling alternatives. If you are using external classes, there's no point in Toggle being a styled component.

File: src/components/Header/styled.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components';
+
+const HeaderPanel = styled.div`
+  background-color: #e93d44;
```
Comment (dianasaurio) [inline src/components/Header/styled.js:4]: I recommend handling the colors with a theme provider, so changing colors with the light and dark theme becomes easier:

https://styled-components.com/docs/advanced#function-themes

File: src/global.css [hunk 6]
```diff
@@ -1,53 +1,8 @@
-html {
-  font-size: 1.125rem;
-  line-height: 1.6;
-  font-weight: 400;
-  font-family: sans-serif;
-  box-sizing: border-box;
-  scroll-behavior: smooth;
-  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+body {
```
Comment (dianasaurio) [inline src/global.css]: You can also handle globals with styled-components: https://styled-components.com/docs/api#createglobalstyle

File: src/components/Header/styled.js (hunk 5 above)
Comment (juanhdezran88) [reply src/components/Header/styled.js:4]: I have planed this, but I will implement it in the next deliverables

**Pull request merged**